base/install_motor.sh

```diff
@@ -34,3 +36,33 @@ cd ..
 
 make -j${JOBS}
 make clean
+
+#Add pmac module
```

Comment:
I don't know if this comment is needed, since we are explicitly talking about pmac right below :thinking: